modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/dashboard-collection/common/components/level-progress-bar/index.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+import {ChartTypes} from '../../enums/chartTypes';
+import {naNToZero} from '../../utils';
+
+const Legend = ({currentValue, total, type}) => {
+	const leftToTotal = Math.floor(total - currentValue);
+
+	const formatLeftToTotal = () => {
+		if (leftToTotal) {
+			if (type === ChartTypes.ARR) {
+				return `$${leftToTotal}k more`;
+			}
+
+			return `${leftToTotal} more project`;
+		}
+
+		return '';
+	};
+
+	const formatValue = (value) => {
+		if (type === ChartTypes.ARR) {
+			return `$${value}K`;
```

Comment:
same here.

Currency and this K need to be dynamic, to add B or M (Millions and Billions). EVP there is something.

Reply:
https://github.com/pt-liferay-solutions/liferay-portal/pull/3553#discussion_r1144752310